pkgs/tools/backup/gyb/default.nix

```diff
@@ -28,14 +27,19 @@ python3Packages.buildPythonApplication rec {
     runHook preInstall
 
     mkdir -p $out/{bin,${python3.sitePackages}}
-    mv gyb.py "$out/bin/gyb"
-    mv *.py "$out/${python3.sitePackages}/"
+    mv gyb.py $out/bin/gyb
+    mv *.py $out/${python3.sitePackages}/
 
     runHook postInstall
   '';
 
   checkPhase = ''
+    runHook preCheck
+
+    unset PYTHONPATH
```

Comment:
Main fix for the build is this. No idea why this is now necessary when it worked before.

Reply:
We should move that into a subshell, to not effect other hooks.

Reply:
Makes sense, changed.